paydb_endpoint.py

```diff
@@ -379,3 +441,33 @@ def transaction_info():
     if tx.user != api_key.user and tx.recipient != api_key.user:
         return bad_request(web_utils.UNAUTHORIZED)
     return jsonify(dict(tx=tx.to_json()))
+
+### TESTING
+@paydb.route('/register_user', methods=['GET', 'POST'])
```

Comment:
what is the purpose of this function?

Reply:
to apply recaptcha to my testing form,

Reply:
if we don't want this, I can delete it off.

Reply:
what is the testing form for?

Reply:
no. the form is the test to get recaptcha in the form. but since we dont need this, ill delete it off